src/ck-core/ckmemcheckpoint.C

```diff
@@ -352,7 +354,8 @@ CkMemCheckPT::CkMemCheckPT(int w)
   if (numnodes  == 1)
 #endif
   {
-    if (CkMyPe() == 0)  CkPrintf("Warning: CkMemCheckPT is disabled due to too few nodes.\n");
+    if (CkMyPe() == 0 && !quietModeRequested)
+      CkPrintf("CharmFT> Warning: In-memory checkpointing is disabled because there are too few nodes.\n");
```

Comment:
Maybe mention explicitly that more than 1 node is needed.

Reply:
Done